src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -151,18 +151,6 @@ span.sidelines {
     border-color: $color-gray-lighter;
   }
 
-  @include media-maxwidth($small-screen) {
```

Comment:
I also want to remove the Foundation imports from Formation. It might be easier to get that in and merged, then make these changes and update package.json in one PR. But it's up to you.